lib/ui/text/paragraph_builder.cc

```diff
@@ -281,9 +281,15 @@ ParagraphBuilder::ParagraphBuilder(
 
   FontCollection& font_collection =
       UIDartState::Current()->window()->client()->GetFontCollection();
-  m_paragraphBuilder = std::make_unique<txt::ParagraphBuilder>(
+
+#if FLUTTER_ENABLE_SKSHAPER
```

Comment:
I'd recommend reducing the amount of divergent code.
```
#if FLUTTER_ENABLE_SKSHAPER
#define FLUTTER_PARAGRAPH_BUILDER txt::ParagraphBuilder::CreateSkiaBuilder
#else
#define FLUTTER_PARAGRAPH_BUILDER txt::ParagraphBuilder::CreateTxtBuilder
#endif
m_paragraphBuilder = FLUTTER_PARAGRAPH_BUILDER(style, font_collection.GetFontCollection());
```

Reply:
done